src/GrumPHP/Process/ProcessBuilder.php

```diff
@@ -72,4 +80,14 @@ private function getCommandLocation($command)
     {
         return $this->externalCommandLocator->locate($command);
     }
+
+    /**
+     * @param Process $process
+     */
+    private function writeVerboseOutput($process)
+    {
+        if ($this->io->isVeryVerbose()) {
+            $this->io->write(PHP_EOL.'Command: '.$process->getCommandLine(), true);
```

Comment:
The last arguments adds a newline at the end of the line, but the symfony ProgressBar class does't add a newline at the end of the task, so without it, we end up with ugly output like in the attached image.

![grumphp](https://cloud.githubusercontent.com/assets/3291952/19493250/c00131f2-952e-11e6-8ddf-a57cf324a571.png)

Another option would be to add the newline to the `$progressFormat` property of `Event\Subscirber\ProgressSubscriber` - in that case, we'd have a blank line between every task, which wasn't something I wanted to do.